src/test/Utils.sol

```diff
@@ -1,6 +0,0 @@
-// SPDX-License-Identifier: UNLICENSED
```

Comment:
Let's bring this code back

Reply:
same as [here](https://github.com/PaulRBerg/prb-math/pull/211#discussion_r1409307455)